python code/ML_batch.py kddcup 3000 5_3_11

train err: 
[0.05291283753694182, 0.02657208788539504, 0.024414858854760073, 0.022029183318199957, 0.021089371743191428, 0.020357764578646326]
attack rate: 
[0.0, 0.0, 0.0, 0.0, 0.0, 0.0]

